layouts/api/list.html

```diff
@@ -126,9 +126,9 @@ <h2 class="mb-2" id="{{ $anchorStr | anchorize }}">
                     <!-- if this is deprecated and we have a higher version link to that -->
                     {{ if (and $endpointVisibility.isVisibleVersion (eq $versionNum "v1") (gt $versionCount 1) (eq (index $tag "x-deprecated") true)) }}}}
                       {{ $collapsedAnchorStr := (print $endpoint.action.summary " " (replace $versionNum "1" "2")) }}
-                      <div class="alert alert-danger">
+                      {{< alert type="danger" >}}
                         <p class="alert-warning">This endpoint is deprecated use the latest endpoint here <a class="toggle-version-tab" href="#{{ $collapsedAnchorStr | anchorize }}">{{- $collapsedAnchorStr -}}.</a></p>
-                      </div>
+                      {{< /alert >}}
```

Comment:
I think this is md shortcode syntax in a html file. I get errors when trying to run this locally